frontend-bugs/src/main/webapp/view/Component.view.xml

```diff
@@ -302,16 +310,6 @@
                                         </l:HorizontalLayout> 
                                     </template>
                                 </Column>
-                                <Column xmlns="sap.ui.table" width="1%" mergeDuplicates="false" 
```

Comment:
Same as above

Reply:
Yes, @serenaponta told me to remove this as it is not used now